stack/global/outputs.tf

```diff
@@ -0,0 +1,7 @@
+output "ip_sets" {
+  value = module.global-accelerator[0].ip_sets
```

Comment:
We need to check if we have a global accelerator enabled, if not, then output a empty string. This will fail without a GA.